.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run tests
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why only run this for the master branch? Same for pull requests against master.

Reply:
Right now we don't have any other branch, do we?

Reply:
Currently, yes, but if you were to ever develop on a different branch, even for pull requests (on this repo), the action would not run until the PR was created.